packages/strapi-provider-email-nodemailer/LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2015-present Strapi Solutions SAS
```

Comment:
I replaced the previous [License](https://github.com/yutikom/strapi-provider-email-nodemailer/blob/master/LICENSE.md) with the one I found [here](https://github.com/strapi/strapi/blob/master/packages/strapi-provider-email-mailgun/LICENSE).